bundle/python_bundle_workflow/README.md

```diff
@@ -0,0 +1,26 @@
+
+# Write a Bundle workflow in Python
+
+MONAI provides the `BundleWorkflow` interface from version 1.2, which is used as the standard API for other applications to access and operate a bundle workflow, like training, evaluation, or inference. `BundleWorkflow` can help reduce the effort of application development and define standard interface and required properties for bundle development. Once a bundle workflow inherits from `BundleWorkflow`, we can accept it in the associated applications, no matter the bundle workflow is implemented by JSON / YAML or python code.
```

Comment:
Maybe we should demonstrate here what keys are required to generate here? Maybe a link for https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/properties.py.